scheduler/src/cook/mesos/api.clj

```diff
@@ -748,7 +749,8 @@
                                mesos-start (:instance/mesos-start-time instance)
                                end (:instance/end-time instance)
                                cancelled (:instance/cancelled instance)
-                               reason (reason/instance-entity->reason-entity db instance)]
+                               reason (reason/instance-entity->reason-entity db instance)
```

Comment:
Should we extract `fetch-instance-map` from `fetch-job-map`? (https://github.com/twosigma/Cook/pull/308/files#diff-6611b39b65ff49438a4d940b51c85cc0R733) That will simplify the testing.